CreeDictionary/API/models.py

```diff
@@ -93,19 +93,25 @@ def get_emoji_for_cree_wordclass(self) -> Optional[str]:
         tags = [FSTTag(t) for t in fst_tag_str.split("+")]
         return LABELS.emoji.get_longest(tags)
 
-    @cached_property
+    # We do our own caching instead of using @cachedproperty so that we can
+    # disambiguate homographs in bulk when displaying search results
```

Comment:
```suggestion
    # We do our own caching instead of using @cachedproperty so that we can
    # disambiguate homographs in bulk when displaying search results
```

👍🏼